angular/projects/spark-angular/src/lib/components/sprk-tabs/sprk-tabs.stories.ts

```diff
@@ -95,8 +143,8 @@ export const defaultStory = () => ({
   `,
 });
 
-defaultStory.story = {
-  name: 'Default',
+deprecated.story = {
+  name: 'Deprecated',
```

Comment:
![image](https://user-images.githubusercontent.com/4342363/104760735-d09d9600-572f-11eb-8a5a-f8edec5902a1.png)
We called this "Legacy (Deprecated)" on Radio